packages/builders/src/jest/schema.json

```
@@ -69,6 +69,11 @@
       "type": "number",
       "description":
         "Max number of workers to run tests across. Useful for CI. (https://jestjs.io/docs/en/cli.html#maxworkers-num)"
+    },
+    "testNamePattern": {
+      "type": "string",
```

Can you add an alias of "t" just like `jest` please? I think this one will be used often and people won't want to write it out. Thanks!

```suggestion
      "type": "string",
      "alias": "t",
```